common/static/coffee/src/discussion/views/discussion_topic_menu_view.js

```diff
@@ -147,44 +144,27 @@
             },
 
             // @TODO move into utils.coffee
-            getNameWidth: function(name) {
-                var test = $('<div>'),
-                    width;
-
-                test.css({
-                    'font-size': this.dropdownButton.css('font-size'),
-                    'opacity': 0,
-                    'position': 'absolute',
-                    'left': -1000,
-                    'top': -1000
-                }).html(name).appendTo(document.body);
-                width = test.width();
-                test.remove();
-                return width;
-            },
-
-            // @TODO move into utils.coffee
             fitName: function(name) {
                 var ellipsisText = gettext('…'),
                     partialName, path, rawName;
 
-                if (this.getNameWidth(name) < this.maxNameWidth) {
+                if (name.length < this.maxNameWidth) {
                     return name;
                 } else {
                     path = _.map(name.split('/'), function(item){
                         return item.replace(/^\s+|\s+$/g, '');
                     });
-                    while (path.length > 1) {
+                    while(path.length > 1) {
                         path.shift();
-                        partialName = ellipsisText + ' / ' + path.join(' / ');
-                        if (this.getNameWidth(partialName) < this.maxNameWidth) {
-                          return partialName;
+                        partialName = ellipsisText + '/' + path.join('/');
+                        if(partialName.length > this.maxNameWidth) {
```

Comment:
should be `if (` with a space (sorry, nitpicky, have just been thinking about linting rules a lot recently...)